File: Old/Removed 2025-01-16/NativeApi/Validator/Validator.cs
#pragma warning disable
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NativeApi.Api
{
    //https://docs.wxwidgets.org/3.2/overview_validator.html
    //https://docs.wxwidgets.org/3.2/classwx_validator.html
    public class Validator
    {
        public static void SuppressBellOnError(bool suppress) { }
        public static bool IsSilent() => default;
    }
}

/*

 */

File: Old/Removed 2025-03-25/Validator.cpp
#include "Validator.h"

namespace Alternet::UI
{
	Validator::Validator()
	{

	}

	Validator::~Validator()
	{
	}

	/*static*/ void Validator::SuppressBellOnError(bool suppress)
	{
		wxValidator::SuppressBellOnError(suppress);
	}

	/*static*/ bool Validator::IsSilent()
	{
		return wxValidator::IsSilent();
	}
}
